ppafm/cpp/Forces.h

```diff
@@ -181,23 +181,23 @@ inline double R4_func(double r2,double &df){
         df     =  0;
         return    0;
     }else{
-        double e = 1 - r2; 
+        double e = 1 - r2;
         df     =  -4*e;
         return     e*e;
     }
 }
 
 inline double invR4_func(double r2,double &df){
     //printf( "invR4_func() \n");
-    double invR2 = 1/r2;
+    double invR2 = 1/(r2 + R2SAFE);
```

Comment:
why you remove R2SAFE ? For precission ?
I guess in practice it does not matter since the atoms never get too close, but anyway ...

Reply:
No, I added it. It was missing and was causing some instability when I was testing the OpenCL implementation against yours.